README.md

```diff
@@ -5,4 +5,50 @@
 [![GitHub release](https://img.shields.io/github/release/lacework/terraform-gcp-audit-log.svg)](https://github.com/lacework/terraform-gcp-audit-log/releases/)
 [![CircleCI status](https://circleci.com/gh/lacework/terraform-gcp-audit-log.svg?style=shield)](https://circleci.com/gh/lacework/terraform-gcp-audit-log)
 
-Terraform module for configuring an integration with Google Cloud Platform Organziations and Projects for Audit Logs analysis.
+Terraform module for configuring an integration with Google Cloud Platform Organizations and Projects for Audit Logs analysis.
+
+
+_**Note:** When `use_existing_service_account` is set to true, the required roles must be added manually. Depending on the integration type this may be at Project or Organization level._
```

Comment:
I am hesitant to change the behavior of all the GCP modules (Config, Audit Log, and GCR),
if we are expecting the users to provide the service account and add the roles manually,
what is the sense of adding this option? If we need to go this route, let us just deprecate it
and have the user use the native Terraform resources for Lacework.

My $0.02 👆🏽

Reply:
Agreed. Kudos to @declanwilson-lw. He pointed me in the direction of using the google_iam_policy data provider to check if the role is already applied when using use_existing_service_account=true